.travis.yml

```diff
@@ -8,7 +8,7 @@ jdk:
 install:
   - sudo apt-get install -y protobuf-compiler
 script:
-  - gradle clean check -x :wallettemplate:check
+  - gradle clean build
```

Comment:
This won't work -- we need to exclude wallettemplate from the build since there is no reliable way to declare the JavaFX dependency.

Reply:
I suggest #2760.

Reply:
Yeah sorry, my local build was using Oracle JDK8.

> there is no reliable way to declare the JavaFX dependency

I might be interested in trying to find a solution for that in the future. There's got to be a way.

> I suggest #2760.

What are you referring to by #2760?

Reply:
No idea where I got this number from. I actually meant  #1711.